.pre-commit-config.yaml

```diff
@@ -35,13 +35,13 @@ repos:
         # we can lint all header files since they aren't "generated" like C files are.
         exclude: ^pandas/_libs/src/(klib|headers)/
         args: [--quiet, '--extensions=c,h', '--headers=h', --recursive, '--filter=-readability/casting,-runtime/int,-build/include_subdir']
--   repo: https://gitlab.com/pycqa/flake8
-    rev: 3.9.2
+-   repo: https://github.com/PyCQA/flake8
+    rev: 4.0.1
     hooks:
     -   id: flake8
         additional_dependencies: &flake8_dependencies
-        - flake8==3.9.2
-        - flake8-comprehensions==3.1.0
+        - flake8==4.0.1
+        - flake8-comprehensions==3.7.0
         - flake8-bugbear==21.3.2
         - pandas-dev-flaker==0.2.0
     -   id: flake8
```

Comment:
looks like flake8 v4 no longer works on Cython files (and that if it did before, it was purely by accident) https://github.com/PyCQA/flake8/issues/1482

@jbrockmendel how important is flake8 linting of cython files? Is it OK to remove it, or would you rather have a separate hook for cython linting which pins flake8 at 3.9.2?

Reply:
id be OK with not having it; i get false-positives about "cimport" all the time

Reply:
sure, it's a hack anyway - @MiTo0o can you remove these cython flake8 checks too please, as well as their config files?

Reply:
Sounds good, I just pushed some changes, lets see if I pass everything and if it meets your expectations.